index.html

```diff
@@ -6,15 +6,66 @@
     <title>Quiz App</title>
     <link rel="stylesheet" href="styles/index.css" />
   </head>
-  <body class="app-grid marginzero">
+  <body class="app-grid">
     <header class="app-grid__header">
       <h1>Quiz App</h1>
     </header>
 
-    <main class="app-grid__main"></main>
-    <section class="main__card">
-      <h2>Question</h2>
-    </section>
+    <main class="app-grid__main">
+      <section class="main__card">
```

Comment:
This section would be it's own BEM-block. 
```suggestion
      <section class="card">
```
Interestingly you names the rest of the card components correctly, starting with `card__something`.